src/main/java/frc/robot/subsystems/shooter/ShooterCal.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.subsystems.shooter;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import frc.robot.Constants;
+
+public class ShooterCal {
+    /** Motor A & B & pivotMotor, Unit Inputs: RPM. output [-1,1] */
```

Comment:
Can you add similar comments for the other sections? So that when people hover over `PIVOT_MOTOR_kI` for example, it'll show the comment